package.json

```diff
@@ -88,6 +89,7 @@
     "eslint": "7.3.0",
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-prettier": "3.1.4",
+    "expect": "^26.1.0",
```

Comment:
Everything we should need is shipped with Jest already. This isn't ideal to add another dependency here, but maybe this is the easiest way to start